openmc/geometry.py

```diff
@@ -257,7 +257,49 @@ def get_all_lattices(self):
                     lattices[cell.fill.id] = cell.fill
 
         return lattices
+    
+    def get_all_surfaces(self):
+        """
+        Return all surfaces used in the geometry
+
+        Returns
+        -------
+        collections.OrderedDict
+            Dictionary mapping lattice IDs to :class:`openmc.Surface` instances
 
+        """
+        surfaces = OrderedDict()
+        
+        for cell in self.get_all_cells().values():
+            self.get_surfaces_from_region(surfaces, cell.region)
+        return surfaces
+    
+    def get_surfaces_from_region(self, surfaces, region):
```

Comment:
Since I can't really imagine a scenario where this would be called directly by the user, I think this method should be a "private" method. To make a Python method "private" (or at least "hidden" when using tab completion in IPython), you simply add a single preceding underscore `_` to the name of the method.